src/include/duckdb/common/enum_util.hpp

```diff
@@ -124,6 +124,8 @@ enum class FilterPropagateResult : uint8_t;
 
 enum class ForeignKeyType : uint8_t;
 
+enum class FramePart : uint8_t;
```

Comment:
I think this can just be local to the accelerator classes, or even just to window segment trees?